azure/datalake/store/core.py

```diff
@@ -717,16 +720,16 @@ def __exit__(self, *args):
         self.close()
 
 
-def _fetch_range(rest, path, start, end, max_attempts=10):
+def _fetch_range(rest, path, start, end, max_attempts=10, stream=False):
```

Comment:
When would we ever want stream to be true? What does this do, since it looks like it is part of the underlying rest module, which we didn't implement here.